metis-validation/metis-validation-service/src/main/java/eu/europeana/validation/service/ClasspathResourceResolver.java

```diff
@@ -26,13 +25,17 @@
   public LSInput resolveResource(String type, String namespaceURI, String publicId, String systemId,
       String baseURI) {
     try {
-      LSInput input = new ClasspathLSInput();
+      //Read file from system
       String fullPath = new File(prefix, systemId).getAbsolutePath();
-      InputStream stream = Files.newInputStream(Paths.get(fullPath));
+      final byte[] bytes = Files.readAllBytes(Paths.get(fullPath));
```

Comment:
Is there a reason to first read it completely into memory completely? Why not use the stream approach that was there originally?

Reply:
Sonarqube complains about not closing the stream, I found this solution only that would make sonarqube complain to go away.